source/index.txt

```diff
@@ -13,5 +13,59 @@ MongoDB C# Driver
    /compatibility
    /issues-and-help
    /faq
-   
 
+Introduction
+------------
+
+Welcome to the documentation site for the official {+driver-long+}. 
+You can add the driver to your application to work with MongoDB in {+language+}. 
+Download it using `NuGet <https://www.nuget.org/>`__, or set up a runnable
```

Comment:
Question: would it be helpful for the NuGet link to bring users to the specific page for the MongoDB Driver?

Reply:
Good call! Linked to the MongoDB driver instead